covidX/settings.py

```diff
@@ -55,11 +55,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(os.getenv("DEBUG_ENV", None))
 
-ALLOWED_HOSTS = [
-    "*",
-    os.getenv("DJANGO_ALLOWED_HOST", "127.0.0.1"),
-    "localhost",
-]
+
+if DEBUG is True:
+    ALLOWED_HOSTS = [os.getenv("DJANGO_ALLOWED_HOST")]
+else: 
```

Comment:
Wouldn't it more readable to write `if DEBUG:` rather than `if DEBUG is True:`, considering `DEBUG` has already been defined as a Boolean?

Reply:
I think they both serve the same purpose. In the case of readability `if DEBUG is True` takes a more behavioral and clearer approach. But I do get where you're coming from, `if DEBUG` is basically a simplified form of the former.

Reply:
I see that it is more of a style choice than anything else. Do you think a one-liner solution (e.g. `ALLOWED_HOSTS = [[os.getenv("DJANGO_ALLOWED_HOST")] if DEBUG else ['0.0.0.0']`) would be better or do you feel that it would compromise readability?

Reply:
In my opinion, yes. This is backed by the KISS(keep it simple) design principle. It's relatively easier to read code in block than using a one-liner.

Reply:
Trailing whitespace. covidX/settings.py#L61 ;
 install requirements_dev pkgs and run black;